UI/nix-update/nix-update.cpp

```diff
@@ -0,0 +1,275 @@
+#include "nix-update.hpp"
+#include "crypto-helpers.hpp"
+#include "nix-update-helpers.hpp"
+#include "obs-app.hpp"
+#include "remote-text.hpp"
+
+#include <util/util.hpp>
+#include <blake2.h>
+
+#include <iostream>
+#include <fstream>
+
+#ifdef BROWSER_AVAILABLE
+#include <browser-panel.hpp>
+#endif
+
+struct QCef;
+extern QCef *cef;
+
+#ifndef MAC_WHATSNEW_URL
+#define MAC_WHATSNEW_URL "https://r2.rodney.io/whatsnew.json"
```

Comment:
The temporary URL https://r2.rodney.io/whatsnew.json is used since it contains the signature in the `X-Signature` header as the real server would.